Исправление русского перевода от Russian Doom Community

PROPHESSOR opened the pull request:
В процессе

MorthimerMcMare committed:
Исправил "Нет звука при подборе топора".

Правда, я считаю, его нужно заменить на что-нибудь менее воинственное.

https://trello.com/c/suyAUi0N/

PROPHESSOR committed:
Добавлен json файл HT перевода карты INTRO

PROPHESSOR committed:
Добавлен скрипт для генерации статистики из JSON файла перевода

PROPHESSOR committed:
Теперь скрипт генерирует красивую таблицу

PROPHESSOR committed:
Свёл первые 35 строк в переводе карты INTRO

PROPHESSOR committed:
Продолжаю сведение перевода в INTRO

PROPHESSOR committed:
Завершил сведение перевода INTRO

PROPHESSOR commented:
![image](https://user-images.githubusercontent.com/19251320/53692417-282ef480-3d98-11e9-9c1e-aabc8e6ac2f7.png)
Лучше так: ```Всему личному составу нужно собраться в Переходе!```

PROPHESSOR commented:
![image](https://user-images.githubusercontent.com/19251320/53692454-de92d980-3d98-11e9-9e63-166c6983c774.png)

PROPHESSOR commented:
Неверный контекст. У него, как раз, есть автокарта, т.к. он тоже военный
![image](https://user-images.githubusercontent.com/19251320/53692461-06823d00-3d99-11e9-97a5-1c67bce2f6cd.png)

PROPHESSOR commented:
Дословный перевод звучит плохо, лучше адаптировать как `я открыл дверь`
![image](https://user-images.githubusercontent.com/19251320/53692469-2a458300-3d99-11e9-92ee-d8a07b03ac1a.png)

PROPHESSOR commented:
**Эту** дверь заклинило
![image](https://user-images.githubusercontent.com/19251320/53692503-e0a96800-3d99-11e9-8b0f-8db503bd2ed7.png)

PROPHESSOR commented:
Лучше "скорее найди ключ"
![image](https://user-images.githubusercontent.com/19251320/53692517-34b44c80-3d9a-11e9-817a-74abd92adf91.png)

PROPHESSOR commented:
Слово `практически` влезает
![image](https://user-images.githubusercontent.com/19251320/53692563-e784aa80-3d9a-11e9-9f1c-6deb1c80a53f.png)

PROPHESSOR commented:
В восточной комнате один лишь Малрас
![image](https://user-images.githubusercontent.com/19251320/53692576-1ef35700-3d9b-11e9-8976-38ae2d2387cd.png)

PROPHESSOR commented:
[Почта]
![image](https://user-images.githubusercontent.com/19251320/53692585-3e8a7f80-3d9b-11e9-9d50-7831d6805bf9.png)

PROPHESSOR commented:
Вообще не похоже, просто достал
![image](https://user-images.githubusercontent.com/19251320/53692613-c96b7a00-3d9b-11e9-8252-e638d6f56e7d.png)

PROPHESSOR committed:
Исправил ошибки перевода карты INTRO

Вплоть до https://github.com/DRRP-Team/DRRP/pull/9#issuecomment-468999033

PROPHESSOR commented:
Исправил баги выше

PROPHESSOR committed:
Исправлена буква ё в шрифте

PROPHESSOR committed:
[INTRO] Исправлено выравнивание текстуры компьютера

[Trello] https://trello.com/c/mFwKtm55

PROPHESSOR committed:
Добавлен README.txt (добавляется в .zip архив)

PROPHESSOR committed:
Теперь ревенанты не оставляют труп

[Trello] https://trello.com/c/8k7p8NT9

PROPHESSOR committed:
Исправлена текстура в INTRO

[Trello] https://trello.com/c/XXy9zGu3

PROPHESSOR committed:
Убран WIP текст с готовых секторов

SEC1
SEC2
SEC5

[Trello] https://trello.com/c/Nwj2ktnn

PROPHESSOR committed:
Локализованы пистолетные патроны

[Trello] https://trello.com/c/66I3sojC

PROPHESSOR committed:
Исправлен спрайт кроватей

[Trello] https://trello.com/c/NwnTpyeF

PROPHESSOR committed:
[XHUB] Теперь все патроны расположены ромбом

[Trello] https://trello.com/c/cVPNRvWJ

PROPHESSOR committed:
[SEC1] Исправлено странное выравнивание текста

[Trello] https://trello.com/c/TcXhFI2j

PROPHESSOR committed:
[SEC1] Исправлен скрипт трансформации

[Trello] https://trello.com/c/fxjh5DWD

PROPHESSOR committed:
[SEC6] Убрана дублирующая линия

[Trello] https://trello.com/c/UkgkSHYA

PROPHESSOR committed:
[SEC6] Исправлен баг с закрытой дверью при повторном прохождении

[Trello] https://trello.com/c/HuHQNvXk

PROPHESSOR committed:
[SEC6] Улучшена сцена смерти морпеха

+ Добавлен перепад напряжения
* Автоматический запуск скрипта
* Дженсен в комнате исчезает после того, как догнал Морпеха

[Trello] https://trello.com/c/vYdKhCmo
[Trello] https://trello.com/c/AuWlhIxn

PROPHESSOR committed:
[TITLEMAP] Убрана ошибка D_SILENCE

[Trello] https://trello.com/c/T7FSP06R

PROPHESSOR committed:
[INTRO] Сделал костыль при подборе топора

[Trello] https://trello.com/c/qHvqER5T

PROPHESSOR committed:
Исправлены LANGUAGE строки, связанные с LevelUP'ом

Убраны %s вставки

PROPHESSOR committed:
Добавлена RPG LevelUp система

[Trello] https://trello.com/c/61QNjMJC

PROPHESSOR committed:
Улучшения Levelup системы

+ Добавлен звук (пока тестовый)
* Теперь отображается правильный уровень

[Trello] https://trello.com/c/61QNjMJC

PROPHESSOR committed:
Улучшение системы LevelUp'ов

+ Добавлены остальные характеристики в структуру Player
* Теперь изначально у игрока не нулевой уровень, а первый

[Trello] https://trello.com/c/61QNjMJC

PROPHESSOR committed:
Костыльное исправление бага с кодировкой

В реальном времени заменяются одиночные пробелы на двойные...
Это баг GZDoom, так что, или менять в файлах, или в реальном времени.
Так же, исправил букву "ё" на "е" в слове "серьёзно ранены"

[Trello] https://trello.com/c/Yjn7YM8b

PROPHESSOR committed:
Улучшение исправления кодировки

Я просто перезаписываю один пробел одним пробелом...
Чёрт возьми, я не знаю, ни как это работает, ни почему это работает
Оно просто работает =D

[Trello] https://trello.com/c/Yjn7YM8b

PROPHESSOR committed:
Заменил все "ё" на "е", уж слишком они багают

[Trello] https://trello.com/c/Yjn7YM8b

PROPHESSOR commented:
@MorthimerMcMare ты там долго? Вон уже конфликты файлов появились!
Кто их решать будет?

MorthimerMcMare reviewed (changes requested):
Заголовок "Про ее":
![](https://media.discordapp.net/attachments/505122055935623199/556483457375666176/unknown.png?width=798&height=426)

Нет запятой после "и" во второй строке.
![](https://media.discordapp.net/attachments/505122055935623199/556491673493045298/unknown.png?width=756&height=426)

Отсутствует точка и перенос строки перед "Увидимся":
![](https://media.discordapp.net/attachments/505122055935623199/556492629777448960/unknown.png?width=755&height=426)

Здесь нет точки в конце:
![](https://media.discordapp.net/attachments/505122055935623199/556493937876008995/unknown.png?width=798&height=426)

По переносам: "Пе-ре-клю-ча-ться" и "Жёл-тую".
![](https://media.discordapp.net/attachments/505122055935623199/556497139669270539/unknown.png?width=756&height=426)
![](https://media.discordapp.net/attachments/505122055935623199/556498046037524521/unknown.png?width=798&height=426)

Нет запятой:
![](https://media.discordapp.net/attachments/505122055935623199/556506701914963968/unknown.png?width=798&height=426)

Обращение к Би Джею должно выделяться запятыми, если здесь не особый, вульгарный стиль общения:
![](https://media.discordapp.net/attachments/505122055935623199/556506815853232139/unknown.png?width=798&height=426)

PROPHESSOR committed:
Исправлены изменения, запрошенные @MorthimerMcMare

[PullRequest] https://github.com/DRRP-Team/DRRP/pull/9

PROPHESSOR commented:
Исправлено.
Отправляю на повторный review.

MorthimerMcMare reviewed (changes requested):
"Та-бель-но-е":
![](https://media.discordapp.net/attachments/332528078733705218/559734938610565120/unknown.png?width=798&height=426)

"Пе-ре-клю-ча-ться":
![](https://media.discordapp.net/attachments/332528078733705218/559746520065376257/unknown.png?width=798&height=426)

"М. Росс", указанный в заголовке письма, своей подписью не светится:
![](https://media.discordapp.net/attachments/332528078733705218/559748120490016770/unknown.png?width=798&height=426)

Опять ошибки с "ё"?
![](https://media.discordapp.net/attachments/332528078733705218/559751131396309033/unknown.png?width=756&height=426)

PROPHESSOR committed:
Исправление, затребованные @MorthimerMcMare

[PullRequest] https://github.com/DRRP-Team/DRRP/pull/9#pullrequestreview-218386092

PROPHESSOR commented:
Исправил всё, кроме подписи Росса...
Я не знаю, как его зовут

PROPHESSOR committed:
Все Проходы заменены на Переходы

PROPHESSOR committed:
Избавление от маразма

Как я понял, перевод затянулся и мы явно не успеем перевести остальные
сектора, поэтому, я прошелся и исправил самые бредовые моменты.
Итак, в ролях:
- Пицтьоц
- Я по колено в смерти
- Марайн
- Проход
- За любовь ко всем святым, чувак! (Вся переписка)
- Оставь терминал, один!
- Циклическая операция двери
- Циклическая конфигурация
- Синтаксис ЯС3 (который, на самом деле, SQL)
- Логин Корня
- вср /мар.дат
- Прочие программистские отсылки
- Прочий быдлоперевод, до которых я успел дотянуться

PROPHESSOR committed:
Вставил пропущенные переносы строк

В ответе Ксавьера в SEC2
